control-plane/pkg/reconciler/trigger/v2/triggerv2.go

```diff
@@ -118,7 +143,21 @@ func (r Reconciler) reconcileConsumerGroup(ctx context.Context, broker *eventing
 		offset = sources.OffsetEarliest
 	}
 
-	newcg := &internalscg.ConsumerGroup{
+	secret, err := security.Secret(ctx, &security.AnnotationsSecretLocator{Annotations: broker.Status.Annotations, Namespace: trigger.Namespace}, r.SecretProviderFunc())
+	if err != nil {
+		return nil, fmt.Errorf("failed to get secret: %w", err)
+	}
+	if err := r.TrackSecret(secret, broker); err != nil {
+		return nil, fmt.Errorf("failed to track secret: %w", err)
+	}
+
+	bootstrapServers := broker.Status.Annotations[kafka.BootstrapServersConfigMapKey]
+	topicName, ok := broker.Annotations[ExternalTopicAnnotation]
+	if !ok {
+		topicName = kafka.BrokerTopic(TopicPrefix, broker)
+	}
```

Comment:
There is a status annotation to read the topic name, see the other reconciler here: https://github.com/knative-sandbox/eventing-kafka-broker/blob/d787a6ebb77e26e8ef759fb1774048ed41655e84/control-plane/pkg/reconciler/trigger/trigger.go#L390-L393